### ISSUE TITLE
Switching from BC to CS

### ISSUE THREAD

**Issue by gus-massa** [greghendershott/travis-racket#38]
Perhaps it would be nice to add a HEADBC, RELEASEBC and RACKET_BC options. I'm not sure if all the URL had stabilized yet.

**Comment by greghendershott** [comment]
I agree, for sure eventually. But I'm inclined to wait to do this, until the Racket release where CS becomes the default. 

Because it seems like the biggest value (and possibly messiest thing, here) will be making HEAD and RELEASE do the default thing (CS or BC) based on how old/new the version of Racket is? (For people who just want a test matrix that covers the default variant for each version.)  It seems it would be easier to figure all that out, after the dust settles?

I admit I don't know if that's "smart lazy" or just "lazy lazy".

**Comment by greghendershott** [comment]
For other people reading this:

Racket's core has been a VM implemented in C.

For a couple versions, there has also been a variant built using Chez Scheme -- but the default is still the old thing.

The idea is eventually the default will become the Chez implementation (and even later, probably the old C VM will be dropped entirely).

The Chez Scheme variant became known as "CS". And the old variant as "BC" (Before Chez).

**Comment by gus-massa** [comment]
I think the lazy lazy version is fine for now.

(Do you want to close this issue for now?)

**Comment by greghendershott** [comment]
I think it's helpful to keep it open.

- The Racket release defaulting to CS is probably less than a year away.
- In the meantime this issue probably has some explanatory value.

**Comment by greghendershott** [comment]
More re this:

https://groups.google.com/forum/#!msg/racket-users/k_o3Ug-lK_s/HwhCtbC6BQAJ

It looks like some of this will be actionable, sooner, >= August 13.

**Comment by greghendershott** [comment]
Just merged commit e290db2 to make "HEAD" and "HEADCS" mean the CS snapshot, which is now the default snapshot. The new "HEADBC" means the BC snapshot.

This doesn't yet address "RELEASE".

It definitely doesn't try to think through what "RACKET_CS" (and a possible new "RACKET_BC"?) would mean for various numbered releases, old and new, someday when the numbered releases start to default to CS.
